src/Yoti/YotiClient.php

```diff
@@ -152,19 +152,22 @@ public function getActivityDetails($encryptedConnectToken = NULL)
         }
 
         // Set remember me Id
-        $rememberMeId = array_key_exists('remember_me_id', $this->_receipt) ? $this->_receipt['remember_me_id'] : NULL;
+        $rememberMeId = $this->parseFromReceipt('remember_me_id');
 
-        // If no profile return empty ActivityDetails object
-        if(empty($this->_receipt['other_party_profile_content']))
+        // Retrieve Human Profile Attribute list
```

Comment:
We're only referring to a "human" profile in the Java SDK aren't we? Are we referring to it a "user profile",and an "application profile" in other SDKs?

Reply:
We do need to have an _ApplicationProfile_.  Then we need a suitable name for the _Human/User Profile_ that clearly differentiates it.  And in SDKs that support OO it seems highly likely they should both inherit from an abstract _Profile_.

I don't mind if you go with _HumanProfile_ or _UserProfile_.  We do want consistency, but can afford a little flexibility in the name of common sense.

I would say that within a given SDK we should be consistent.  Once we've chosen one or the other we'll still with it - changing it and forcing users to fix their code seems unnecessary to me.

Maybe if we want to avoid the renaming of _Profile_ we could introduce an abstract _BaseProfile_ where necessary.  I know that some SDKs have only recently renamed the _Profile_ object - let's not antagonise our SDK users by switching straight back.